lib/exporter/EnvFileExporter_1_0_0.js

```diff
@@ -53,15 +52,15 @@ class EnvFileExporter_1_0_0 {
       (next) => {
         this.applicationsService.getByIdOrName(env.app, (err, app) => {
           if (err) return next(err);
-          self.servicesService.getAllByDomainType(DomainTypes.APP, { id: env.app }, (errAppServices, appServices) => {
+          this.servicesService.getAllByDomainType(DomainTypes.APP, { id: env.app }, (errAppServices, appServices) => {
```

Comment:
I'm probably missing something but why do we need all the services?

Reply:
We need them for the export of custom endpoints and collection hooks. There we only have the ID of the backing server that handles the endpoint/hook and we want to export the name of the service. In order to get the name of the service, we need to get all available services and then search the one with the specific backing server ID. So, in order to do that I get all the services once, in the beginning, instead of getting them for every endpoint/hook I want to export.